release/nightly_tests/dataset/pipelined_ingestion_compute.yaml

```diff
@@ -0,0 +1,29 @@
+cloud_id: cld_17WvYIBBkdgLwEUNcLeRAE
+region: us-west-2
+
+max_workers: 999
+
+aws:
+    IamInstanceProfile: {"Name": "ray-autoscaler-v1"}
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: 500
+
+head_node_type:
+    name: head_node
+    instance_type: i3.8xlarge
```

Comment:
Is i3 necessary here? (what about just using m5?)

Reply:
yeah we need at least 70GB plasma store, which translate 256GB memory for some reason...